### PR TITLE
fix german translation

### PR DESCRIPTION
**Pull request by jens1o**
I'd missed a suffix...

### CODE REVIEW

**Comment by JimiC** [comment]
It's highly recommended that all PRs are from a separate `branch` and not from `master`.

**Comment by jens1o** [comment]
Oops, I didn't looked this time after the branch.

**Comment by JimiC** [comment]
@robertohuertasm You may want to update `README` adding a section about the available translation and ask for help for the remaining languages.

**Comment by JimiC** [comment]
Please also fix line 34. Linting is failing.

File: src/i18n/langResources/lang.de.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export const langDe: ILangResource = {
   newVersionMessage: 'Willkommen zur neuen Version von vscode-icons!',
   seeReleaseNotes: 'Was wurde geändert? (Englisch)',
   dontShowThis: 'Diese Nachricht beim nächsten Update nicht mehr anzeigen',
-  seeReadme: 'Etwas über die Erweiterung lernen',
+  seeReadme: 'Etwas über die Erweiterung lernen (Englisch)',
   welcomeMessageBegin: 'vscode-icons benutzt nun die offizielle API. Aktiviere die Icons unter ',
   welcomeMessageEnd: ' und wähle VSCode Icons aus.',
```
Comment (JimiC) [inline src/i18n/langResources/lang.de.ts]: Remove beginning whitespace.

Comment (JimiC) [inline src/i18n/langResources/lang.de.ts]: Generally, fix the excess whitespaces in messages you have introduced.

Comment (jens1o) [inline src/i18n/langResources/lang.de.ts]: I've oriented myself with the english file, and there are also whitespaces?

Comment (JimiC) [inline src/i18n/langResources/lang.de.ts]: Check `en` version of `welcomeMessageEnd` again. And it's not only there.

**Comment by jens1o** [comment]
like this?

**Comment by JimiC** [comment]
Also you use `Starte neu` in some messages when you should use `\'Neu starten\'` as we are referencing the `Restart` button.

**Comment by jens1o** [comment]
But it's really bad german to say so, that's why I describe it indirectly. In German, you wouldn't say so.

**Comment by JimiC** [comment]
We are saying `Restart, for changes to take effect` but merely `Press 'Restart', for changes to take effect`.

**Comment by jens1o** [comment]
According to me...

**Comment by JimiC** [comment]
Then change the `reload` text to `Starten neu`.

**Comment by jens1o** [comment]
That's also bad German. The new phrase is ok. Not the best, not the worst.

**Comment by JimiC** [comment]
It's too long now.

**Comment by JimiC** [comment]
Well, never mind.

**Comment by jens1o** [comment]
Wait, I missed a `'`

**Comment by JimiC** [comment]
Please fix the linting issue and we are done.

**Pull request merged**